data/CMakeLists.txt

```diff
@@ -87,53 +87,35 @@ IF(WIN32)
     PROJECT(VISIT_DATA)
 ENDIF(WIN32)
 
-MESSAGE(STATUS "Configuring VisIt Binary Tarball Data Targets")
+MESSAGE(STATUS "Configuring VisIt Binary Data Archive Targets")
 
-#-----------------------------------------------------------------------------
-# Set up archiver executable name (7z, 7za and p7zip are common ones)
-#-----------------------------------------------------------------------------
-IF(NOT VISIT_DATA_ARCHIVER_NAME)
-    list(APPEND VISIT_DATA_ARCHIVER_NAME "7z" "7za" "7zr" "p7zip")
-ENDIF()
+FIND_PACKAGE(Python3)
```

Comment:
You should use the python executable found by the FindVisItPython.cmake. Besides which, this will only work if you want to use python3 installed in standard location.

Reply:
If I do it that way, it won't work for me or @JustinPrivitera as niether of us have a VisIt installed python that has the needed modules (see introductory remarks about this). We want/need to use the default python.

Also, I don't want cmaking in this dir to depend on anything in VisIt. It shouldn't have to.

Any other ideas?